### PR TITLE
Add pr test against komodo

### PR DESCRIPTION
**Pull request by ManInFez**
Now that ert-storage has been introduced to komodo, we can have a ci-pipeline running the ert-storage tests with the libraries included in komodo.

### CODE REVIEW

**Comment by dotfloat** [comment]
Is it possible to test this without merging? Also, you have a typo in the commit summary.

**Comment by ManInFez** [comment]
tests are already running on this PR

**Comment by ManInFez** [comment]
link to jenkins-job: https://ci.equinor.com/scout/job/komodo_testing/job/ert-storage-pr/

File: ci/jenkins/testkomodo.sh [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+install_test_dependencies () {
+    pip install .[test]
+}
+
+install_package () {
+    pip install . --no-deps
+}
+
+start_tests () {
```
Comment (xjules) [inline ci/jenkins/testkomodo.sh:14]: Just wondering why not mypy test?

**Review by oysteoh — APPROVED** [review]
🥓

**Pull request merged**